backend/src/main/java/capstone/storage/backend/item/ItemController.java

```diff
@@ -0,0 +1,57 @@
+package capstone.storage.backend.item;
+
+import capstone.storage.backend.exceptions.ExceptionMessage;
+import capstone.storage.backend.exceptions.IsNullOrEmptyException;
+import capstone.storage.backend.exceptions.ItemForbiddenRequestException;
+import capstone.storage.backend.exceptions.ItemToDeleteNotFoundException;
+import capstone.storage.backend.item.models.AddItemDto;
+import capstone.storage.backend.item.models.Item;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/items/")
+@RequiredArgsConstructor
+public class ItemController {
+    private final ItemService service;
+
+    @GetMapping
+    public List<Item> getAllItems() {
+        return service.findAll();
+    }
+
+    @PostMapping(value = {"{eanToFind}", ""})
```

Comment:
kürzere Schreibweise möglich
```suggestion
    @PostMapping({"{eanToFind}", ""})
```